build.gradle.kts

```diff
@@ -350,6 +351,7 @@ subprojects {
         "UnnecessaryTypeArgument",
         "UnusedAnonymousClass",
         "UnusedCollectionModifiedInPlace",
+        "UnusedVariable",
```

Comment:
Could you let the Error-prone code follow the alphabet order? Such as `EqualsGetClass`, `DefaultCharset`, `InlineFormatString`.

Reply:
Done